### PR TITLE
Add support for traveltimesdf

### PR DESCRIPTION
**Pull request by yeesian**
It was dropped in https://github.com/yeesian/SchedulingModels.jl/pull/155 and should be supported.

### CODE REVIEW

**Comment by juliayyan** [comment]
I can't load TransitNetworks, can you?

```
[ Info: Precompiling TransitNetworks [dd8af4cf-a4c1-5c74-ad1c-62dfd815ddef]
ERROR: LoadError: LoadError: UndefVarError: segments not defined
Stacktrace:
 [1] top-level scope at none:0
 [2] include at .\boot.jl:317 [inlined]
 [3] include_relative(::Module, ::String) at .\loading.jl:1038
 [4] include at .\sysimg.jl:29 [inlined]
 [5] include(::String) at C:\Users\Julia\.julia\dev\TransitNetworks\src\TransitNetworks.jl:1
 [6] top-level scope at none:0
 [7] include at .\boot.jl:317 [inlined]
 [8] include_relative(::Module, ::String) at .\loading.jl:1038
 [9] include(::Module, ::String) at .\sysimg.jl:29
 [10] top-level scope at none:2
 [11] eval at .\boot.jl:319 [inlined]
 [12] eval(::Expr) at .\client.jl:399
 [13] top-level scope at .\none:3
in expression starting at C:\Users\Julia\.julia\dev\TransitNetworks\src\network\networkproblem.jl:165
in expression starting at C:\Users\Julia\.julia\dev\TransitNetworks\src\TransitNetworks.jl:8
```

This error is a mystery to me because segments seems to be properly defined in the argument...

File: src/network/networkproblem.jl [hunk 1]
```diff
@@ -133,13 +134,33 @@ function TransitNetworkProblem(
...
+    traveltimes = nothing
+    if DataFrames.nrow(traveltimesdf) > 0
+        traveltimes = [zeros(Int, length(stnsegments[seg])) for seg in 1:nsegments]
+        for s in 1:nsegments 
+            linename = pathlist[findfirst(linesegments[:,s])][1,:line] 
+            for i=2:length(stnsegments[s])
+                origin = stns[stnsegments[s][i-1]]
+                destination = stns[stnsegments[s][i]]
+                traveltimes[s][i] = traveltimes[s][i-1] + round.(Int,
+                    Statistics.median(
```
Comment (juliayyan) [inline src/network/networkproblem.jl]: I think you need to add Statistics to TransitNetworks.jl?

Comment (yeesian) [reply src/network/networkproblem.jl]: okay just pushed that change for you to try while I try and figure out what's wrong with the setup on my laptop

**Comment by yeesian** [comment]
I think this should work now.

**Comment by juliayyan** [comment]
yes! good to merge on my end

**Pull request merged**